master_layer/Waypoint waypoint
float64 max_forward_speed
string interpolator
---
bool success
